src/commands/fun/blindfoldedroosen.ts

```diff
@@ -0,0 +1,20 @@
+import { Message } from 'discord.js';
+import { Command } from '@lib/types/Command';
+
+export default class extends Command {
+
+	description = `You've been challenged to a sword fight. However, your opponent, Professor Roosen, has decided to wear a blindfold. Feeling lucky?`;
+	usage = '';
```

Comment:
Oh sorry, I meant that you don't need the usage at all. It's an optional field in a `Command`